protocol/0030-ETHM-multisig_control_spec.md

```diff
@@ -127,3 +127,48 @@ contract MultisigControl {
 ### MultisigControl Consuming Smart Contract
 * Consuming smart contract calls `verify_signatures` with valid signature bundle and message hash is returned true if the valid signature count is over threshold % of total signers  (<a name="0030-ETHM-023" href="#0030-ETHM-023">0030-ETHM-023</a>)
  
+## Multisig Control to Vega Integration Tests (Vega System Tests)
+1. `mapping(address => bool) public signers;`
+   * must show "True" for each signer that is currently valid (<a name="0030-ETHM-024" href="#0030-ETHM-024">0030-ETHM-024</a>)
```

Comment:
What is the meaning of signer being valid from Vega chain side? That they're one of the "consensus forming" validators?

Reply:
yes, that the validators that core knows about return `True` when the contract signers are queried